### PR TITLE
task: Add MLT jobs and update hardcoded values for cronjob-logs

### PR DESCRIPTION
**Pull request by o11yguru**
### Any background context you want to provide for reviewers?
Add MLT jobs and update hardcoded values for cronjob-logs

### CODE REVIEW

File: jobs/job-metrics.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: demo-log-job
```
Comment (arcanez) [inline jobs/job-metrics.yaml:4]:
```suggestion
  name: demo-metric-job
```

File: jobs/job-traces.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: demo-log-job
```
Comment (arcanez) [inline jobs/job-traces.yaml:4]:
```suggestion
  name: demo-trace-job
```

**Review by arcanez — CHANGES_REQUESTED** [review]
how are `jobs/job-*.yaml` different from `jobs/cronjob-*.yaml`?

**Comment by o11yguru** [comment]
> how are `jobs/job-*.yaml` different from `jobs/cronjob-*.yaml`?

We may sometimes want to send data once and only once vs. continuously

**Review by arcanez — APPROVED** [review]

**Pull request merged**